src/js/video.js

```diff
@@ -62,7 +62,7 @@ var videojs = function(id, options, ready){
     }
 
     // If a player instance has already been created for this ID return it.
-    if (Player.players[id]) {
+    if (videojs.getPlayers()[id]) {
```

Comment:
I'm not sure I follow-- `videojs` is defined in this file on line 52. Am I missing something?